tests/base/test_status.py

```diff
@@ -29,9 +29,9 @@ def test_status_codes(self):
             "StatusCode.kUnauthenticated", 
         ]
         actual_codes = []
-        x = range(17)
-        for n in x:
-            actual_codes.append(str(pd.Status.StatusCode(n)))
+        
+        
+        [actual_codes.append(str(pd.Status.StatusCode(n))) for n in range(17)]
```

Comment:
Is there any reason for using generator?
If I print `actual_codes ` after running`[actual_codes.append(str(pd.Status.StatusCode(n))) for n in range(17)]`, I get a generator object. Just curious does it give any advantage?

Reply:
Not really, we performance don't matter here until it will ...
So, for now, it should be fine.

Reply:
Then let's discard this change. It reduces the readability of the code.

Reply:
yup